pantheon-bundle/frontend/src/app/assemblyDisplay.tsx

```diff
@@ -254,6 +254,10 @@ class AssemblyDisplay extends Component<any, any, any> {
 
     private onPublishEvent = () => {
         this.getVersionUUID(this.props.location.pathname)
+        // wait for 1 second so that JCR data gets updated
```

Comment:
assemblyDisplay doesn't have issue with  ""View on Customer Portal" link when publishing. This code is not neccessary.

Reply:
I am seeing this issue on QA for assembly when I am publishing the assembly for the first time.